### ISSUE TITLE
The MIPS environment variables are incorrectly configured (Windows)

### ISSUE THREAD

**Issue by WandererLain** [grumpycoders/pcsx-redux#1230]
### Describe the bug

When [installing a pre-built MIPS toolchain](https://github.com/grumpycoders/pcsx-redux/blob/main/mips.ps1), it's installed in the directory you are currently in, even symlinked into it, but the environment variables are pointed to `%appdata%\mips` where only the installation files and an empty "versions" folder are located.

![Screenshot](https://user-images.githubusercontent.com/113742788/221712302-aa5c38a2-5d89-4a34-8e46-9ecb044ba6e6.png)

### Expected behavior

Install it in %appdata% directly or make the symlink in this folder instead of the current one to match the environment variables.

### Steps to reproduce the bug

1. `powershell -c "& { iwr https://raw.githubusercontent.com/grumpycoders/pcsx-redux/main/mips.ps1 | iex }"`
2. `mips`
3. Select version
4. Wait for installation

### Operating System

Windows 11 Version 22H2

### PCSX-Redux version

Not related to pcsx-redux, only MIPS toolchain setup

### CPU model

AMD Ryzen 7 3750H

### GPU model & Drivers

NVIDIA GeForce GTX 1650

### BIOS version

Not related to pcsx-redux, only MIPS toolchain setup

### Options

- [ ] Dynarec CPU
- [ ] 8MB
- [ ] OpenGL GPU
- [ ] Fastboot
- [ ] Debugger

**Comment by nicolasnoble** [comment]
Huh, installing in the %APPDATA% folder *is* supposed to be the normal behavior. See the directory structures on my machine:

![image](https://user-images.githubusercontent.com/7281574/221714240-eadb662a-c426-4162-84ad-5528d0d61cf8.png)
![image](https://user-images.githubusercontent.com/7281574/221714257-2c407074-e11b-4b18-b6f3-4485afcb300c.png)

with the `mips` link pointing to `versions/12.2.0` in my case. I don't know why on your machine it decided to dump it on the cwd, but that's definitely not what's supposed to happen...

You can see the logic around here:

https://github.com/grumpycoders/pcsx-redux/blob/main/mips.ps1#L223-L225

Basically, it computes the path to itself, and builds the `$symlink` variable, pointing at the absolute path to where the symlink has to be created / updated, and the `$VersionsPath` variable, pointing at the container folder for all the versions you can switch to.

Something went wrong for you where the `$me` variable ended up pointing where you were...?

**Comment by WhimsicalWanderer** [comment]
Yeah, that's weird. I just followed this [MIPS toolchain setup instructions](https://github.com/ABelliqueux/nolibgs_hello_worlds#mips-toolchain-setup). I'm trying to figure out why 🤔

**Comment by WhimsicalWanderer** [comment]
I tested it again in a virtual machine. I see.

If I run the command from the Powershell itself the download is done in the current directory
![image](https://user-images.githubusercontent.com/113742788/221719165-30f53d29-8451-4a5f-88cc-8207941f1506.png)

But from a CMD it is done correctly in %appdata%.
![image](https://user-images.githubusercontent.com/113742788/221718875-5de19929-e0a5-4cc1-af2a-ec9aaef1a792.png)

**Comment by nicolasnoble** [comment]
Ah ha, interesting... I'll try reproducing this locally. Thanks for the reproduction pointer, this is helpful.
